### PR TITLE
First cut at talking about clusters and partitions

### PR DESCRIPTION
**Pull request by jbednasz**

### CODE REVIEW

File: pages/hpc/clusters.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-# Clusters/Partitions
+# Clusters
 
 CCR maintains two clusters; *ub-hpc* and *faculty*. Within each cluster exist partitions (queues) that users can request to have their jobs run on. Not all partitions are available to all users. Below are description of the clusters and the partitions.
```
Comment (dsajdak) [inline pages/hpc/clusters.md:3]:
```suggestion
CCR maintains two clusters; *ub-hpc* and *faculty*. Within each cluster exist partitions (queues) that users can request to have their jobs run on. Not all partitions are available to all users. Below are descriptions of the clusters and the partitions. 
```

File: pages/hpc/clusters.md [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
 
-## Industry 
+The ub-hpc cluster contains the following partitions:
 
-Write me!
+* debug: Dedicated compute nodes to help users debug their workflows  
+* general-compute: Large pool of compute nodes for academic users
+* viz: Hardware Accelerated Graphics compute nodes
+* industry: Compute nodes industry partners pay to use
+* scavenger: Peemptible jobs on all ub-hpc nodes
```
Comment (dsajdak) [inline pages/hpc/clusters.md:17]:
```suggestion
* scavenger: Preemptible jobs on all ub-hpc nodes
```

File: pages/hpc/jobs.md [hunk 3]
```diff
@@ -148,6 +148,10 @@ echo "Hello world from faculty cluster node: "`/usr/bin/uname -n`
 #Let's finish some work
 
 ```
+!!! Note 
+	Jobs on the faculty cluster are allowed to run up until the downtime starts. Please ensure your jobs checkpoint and can restart where they left off OR request only enough time to run your job prior to the 7am cutoff.
+
```
Comment (dsajdak) [inline pages/hpc/jobs.md:153]: We know this refers to the maintenance downtimes but it's not clear when reading from top to bottom.  Recommend something like:
```suggestion
!!! Note "Caution: Maintenance Downtimes"
    Jobs on the faculty cluster are allowed to run up until the downtime starts. Please ensure your jobs checkpoint and can restart where they left off OR request only enough time to run your job prior to the 7am cutoff on maintenance days.  See the [schedule here](https://ubccr.freshdesk.com/support/discussions/forums/5000296650)
```

**Review by dsajdak — CHANGES_REQUESTED** [review]
Just a couple minor suggestions.  Otherwise looks good!

**Comment by jbednasz** [comment]
All accepted.

**Review by dsajdak — APPROVED** [review]

**Pull request merged**